packages/flutter/test/rendering/editable_test.dart

```diff
@@ -134,10 +134,12 @@ void main() {
         offset: 0,
       ),
     );
-    editable.layout(BoxConstraints.loose(const Size(1000.0, 1000.0)));
+    layout(editable, constraints: BoxConstraints.loose(const Size(500.0, 500.0)));
```

Comment:
Why did you change this size?  I admit I don't think I've looked at these tests before.